docs/source/commandline/sdlconfig.rst

```diff
@@ -5,375 +5,182 @@ SDL-Specific Commandline Options
 
-**-[no]syncrefresh**
-
-	Enables speed throttling only to the refresh of your monitor. This means that the game's actual refresh rate is ignored; however, the sound code still attempts to keep up with the game's original refresh rate, so you may encounter sound problems. This option is intended mainly for those who have tweaked their video card's settings to provide carefully matched refresh rate options. The default is OFF (*-nosyncrefresh*).
-
-
-Video soft-specific options
+Video Soft-Specific Options
 ---------------------------
 
+.. _mame-scommandline-scalemode:
+
 **-scalemode**
 
 	Scale mode: none, async, yv12, yuy2, yv12x2, yuy2x2 (**-video soft** only). Default is '*none*'.
 
 
 
-Video OpenGL-specific options
------------------------------
-
-**-[no]filter** / **-[no]flt**
-
-	Enable bilinear filtering on the game screen graphics. When disabled, point filtering is applied, which is crisper but leads to scaling artifacts. If you don't like the filtered look, you are probably better off increasing the **-prescale** value rather than turning off filtering altogether. The default is ON (*-filter*).
-
-**-prescale** *<amount>*
-
-	Controls the size of the screen images when they are passed off to the graphics system for scaling. At the minimum setting of 1, the screen is rendered at its original resolution before being scaled. At higher settings, the screen is expanded by a factor of *<amount>* before being scaled. This produces a less blurry image at the expense of some speed and also increases the effective resolution of non-screen elements such as artwork and fonts. The default is *1*.
-
-Video OpenGL debugging options
+Video OpenGL Debugging Options
 ------------------------------
 	
 These 4 options are for compatibility in **-video opengl**.  If you report rendering artifacts you may be asked to try messing with them by the devs, but normally they should be left at their defaults which results in the best possible video performance.	
 
+.. _mame-scommandline-glforcepow2texture:
+
```

Comment:
The OpenGL render module isn't specific to the SDL OSD any more - it's compiled in by default on Windows.  Are these options really not present in Windows builds?  I was under the impression that GLSL filters are pretty popular with Windows users.

Reply:
Wasn't aware OpenGL was included on Windows now, but outside of BGFX shaders the preferred shader on the Windows side would be HLSL.

I'll need to rework the video subsection to get that corrected.

Reply:
Not necessarily.  The trouble is the D3D9 HLSL feature can only really use the supplied HLSL shaders, while OpenGL can chain several user-supplied shaders.  It's also easier to get started with if you're experimenting because you don't need to build the bgfx tools to compile your shaders, you can just drop them and they get compiled when they're loaded.  There are definitely some use cases for using GLSL filters.